.env_acceptance.dist

```diff
@@ -1,7 +0,0 @@
-###> symfony/framework-bundle ###
-APP_ENV=prod
-APP_DEBUG=0
-APP_SECRET=67d829bf61dc5f87a71fd814e2c9f629
-###< symfony/framework-bundle ###
-
-DEBUG_COOKIE=
```

Comment:
This file is needed for acceptance test show production env https://github.com/rectorphp/getrector.org/blob/558aca6b5b71a62d38955cc1eb5ef48fe1e30fdd/.github/workflows/acceptance_test.yaml#L25

Reply:
Fixed